src/components/structural/header/Header.js

```diff
@@ -52,8 +54,18 @@ class Header extends Component {
             referenceOpen: false,
             coursesOpen: false,
             tourOpen: false,
-            welcomeOpen: false
+            welcomeOpen: false,
+            updateCollect: false,
+            fetchCollect: false,
```

Comment:
Can we write out the full word "collection"? When I think of collect I think of the verb.